src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.scss

```diff
@@ -11,3 +11,18 @@
     justify-content: space-between;
   }
 }
+
+.noFriends-wraper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  .noFriends {
+    font-family: 'Montserrat', sans-serif;
```

Comment:
do not hardcode font family, use a global variable or create one

Reply:
fixed